content/en/docs/tasks/observability/metrics/classify-metrics/index.md

```diff
@@ -0,0 +1,351 @@
+                            },
+                            {
+                                "value": "503",
+                                "condition": "response.code == 503"
+                            },
+                            {
+                                "value": "5xx"
+                                "condition": "response.code >= 500 && response.code <= 599"
+                            },
+                            {
+                                "value": "4xx",
+                                "condition": "response.code >= 400 && response.code <= 499"
+                            }
+                            ]
+                        }
+                        ]
+                    }
+                    vm_config:
+                    runtime: envoy.wasm.runtime.null
+                    code:
+                        local: { inline_string: "envoy.wasm.attributegen" }
+    {{< /text >}}
+
+1. Apply your changes using the following command:
+
+    {{< text bash >}}
+    $ kubectl -n istio-system apply -f attribute_gen_service.yaml
+    {{< /text >}}
+
+1. Find the `stats-filter-1.6` `EnvoyFilter` resource from the `istio-system`
+   namespace, using the following command:
+
+    {{< text bash >}}
+    $ kubectl -n istio-system get envoyfilter | grep ^stats-filter-1.6
+    stats-filter-1.6                    2d
+    {{< /text >}}
+
+1. Create a local file system copy of the `EnvoyFilter` configuration, using the
+   following command:
+
+    {{< text bash >}}
+    $ kubectl -n istio-system get envoyfilter stats-filter-1.6 -o yaml > stats-filter-1.6.yaml
+    {{< /text >}}
+
+1. Open `stats-filter-1.6.yaml` with a text editor and locate the
+   `envoy.wasm.stats` extension configuration. The default configuration is in
+   the `configuration` section like this example:
+
+    {{< text json >}}
+    {
+    "debug": "false",
+    "stat_prefix": "istio"
+    }
+    {{< /text >}}
+
+1. Edit the configuration section for each instance of the extension
+   configuration. For example, to update `response_code` and add
+   `request_operation` dimensions to the standard `requests_total` metric,
+   change it like this example:
+
+    {{< text json >}}
+    {
+    "debug": "false",
+    "stat_prefix": "istio",
+    "metrics": [
+        {
+            "name": "requests_total",
+            "dimensions": {
+            "response_code": "has(istio.responseClass)?istio.responseClass:response.code",
+            "response_operation": "istio.operationId",
+            }
+        }
+    ]
+    }
+    {{< /text >}}
+
+1. Save `stats-filter-1.6.yaml` and then apply the configuration using the following command:
+
+    {{< text bash >}}
+    $ kubectl -n istio-system apply -f stats-filter-1.6.yaml
+    {{< /text >}}
+
+1. Generate metrics by sending traffic to your application.
+
+1. Visit Prometheus and look for the new or changed dimensions, for example
+   `2xx`.
```

Comment:
This block is similar to "Verify the results" block?

Reply:
Fixed